src/pages/hegel/reference/the-mechanical-process/the-formal-mechanical-process-section-1.mdx

```diff
@@ -0,0 +1,243 @@
+---
+title: The Formal Mechanical Process, Section 1
+description:
+    Learn about the first section of the Formal Mechanical Process from Hegel's
+    Science of Logic
+isArticle: true
+authors: Ahilleas Rokni (2024)
+---
+
+## (a) The Formal Mechanical Process: Section 1
+
+The first section of B. The Mechanical Process that deals with the logical
```

Comment:
This reads very clunky and heavy-handed. Perhaps start with "The mechanical process in Hegel is divided into three seperate sections: " and then list the sections/phases/groupings, and state that you'll deal with "section X" in this article, or something.